cms/src/main/java/uk/nhs/digital/externalstorage/workflow/action/ExternalFilePublishAction.java

```diff
@@ -0,0 +1,34 @@
+package uk.nhs.digital.externalstorage.workflow.action;
+
+import org.apache.commons.scxml2.SCXMLExpressionException;
+import org.apache.commons.scxml2.model.ModelException;
+import org.onehippo.cms7.services.HippoServiceRegistry;
+import org.onehippo.repository.documentworkflow.action.AbstractDocumentTaskAction;
+
+import uk.nhs.digital.externalstorage.s3.S3Connector;
+import uk.nhs.digital.externalstorage.workflow.task.ExternalFilePublishTask;
+
+public class ExternalFilePublishAction extends AbstractDocumentTaskAction<ExternalFilePublishTask> {
+
+    private static final long serialVersionUID = 1L;
```

Comment:
Why do you need this?

Reply:
copy and paste from Minos code - removing.